### ISSUE TITLE
Upgrade qunitjs to 2.x

### ISSUE THREAD

**Issue by mucaho** [gruntjs/grunt-contrib-qunit#122]
As of this writing, `qunitjs@2.0.0-rc1` is available.
The new major version introduces breaking changes across the board, see [migration guide](http://qunitjs.com/upgrade-guide-2.x/).

**Comment by Arkni** [comment]
Let's wait for `2.0.0` to be released.

**Comment by jzaefferer** [comment]
Or put together a PR with a release candidate. If something breaks, report back to QUnit. If not, wait for the final release, update the PR once more and merge it.

@mucaho would you work on that?

**Comment by mucaho** [comment]
@jzaefferer It seems like there is minimal coupling with `qunitjs`, which should make this transition pretty painless.
Afaik, `phantomjs/bridge.js` and the `test/`s would be prime candidates for regressions.

Here is a preliminary patch that fixes those issues and sets `qunitjs` to `2.0.0-rc1`.

Any copyright of the following patch is dedicated to the Public Domain.
http://creativecommons.org/publicdomain/zero/1.0/
```patch
diff --git a/package.json b/package.json
index 68b14e9..d164e8a 100644
--- a/package.json
+++ b/package.json
@@ -24,7 +24,7 @@
     "grunt-contrib-connect": "^1.0.0",
     "grunt-contrib-internal": "^1.1.0",
     "grunt-contrib-jshint": "^1.0.0",
-    "qunitjs": "^1.20.0"
+    "qunitjs": "2.0.0-rc1"
   },
   "keywords": [
     "gruntplugin"
diff --git a/test/qunit_modules.js b/test/qunit_modules.js
index 87fb36d..2081d1a 100644
--- a/test/qunit_modules.js
+++ b/test/qunit_modules.js
@@ -1,14 +1,14 @@
-module('module1');
+QUnit.module('module1');
 
-test('basic test', function() {
-  expect(1);
-  ok(true, 'this had better work.');
+QUnit.test('basic test', function(assert) {
+  assert.expect(1);
+  assert.ok(true, 'this had better work.');
 });
 
 
-module('module2');
+QUnit.module('module2');
 
-test('a second basic test', function() {
-  expect(1);
-  ok(true, 'this had better work.');
+QUnit.test('a second basic test', function(assert) {
+  assert.expect(1);
+  assert.ok(true, 'this had better work.');
 });
diff --git a/test/qunit_noglobal_test.js b/test/qunit_noglobal_test.js
index 77f35e8..69b2964 100644
--- a/test/qunit_noglobal_test.js
+++ b/test/qunit_noglobal_test.js
@@ -2,7 +2,7 @@
 var pushFailure = QUnit.pushFailure;
 var failures = [];
 
-test('global pollution', function(assert) {
+QUnit.test('global pollution', function(assert) {
   window.pollute = true;
   assert.ok(pollute, 'nasty pollution');
   QUnit.pushFailure = function(message) {
@@ -10,8 +10,8 @@ test('global pollution', function(assert) {
   };
 });
 
-test('actually check for global pollution', function() {
-  expect(1);
+QUnit.test('actually check for global pollution', function(assert) {
+  assert.expect(1);
   QUnit.pushFailure = pushFailure;
-  deepEqual(failures, ['Introduced global variable(s): pollute']);
+  assert.deepEqual(failures, ['Introduced global variable(s): pollute']);
 });
diff --git a/test/qunit_test.js b/test/qunit_test.js
index e6a5ca7..28c01a7 100644
--- a/test/qunit_test.js
+++ b/test/qunit_test.js
@@ -1,12 +1,12 @@
 
-test('basic test', function() {
-  expect(1);
-  ok(true, 'this had better work.');
+QUnit.test('basic test', function(assert) {
+  assert.expect(1);
+  assert.ok(true, 'this had better work.');
 });
 
 
-test('can access the DOM', function() {
-  expect(1);
+QUnit.test('can access the DOM', function(assert) {
+  assert.expect(1);
   var fixture = document.getElementById('qunit-fixture');
-  equal(fixture.innerText, 'this had better work.', 'should be able to access the DOM.');
+  assert.equal(fixture.innerText, 'this had better work.', 'should be able to access the DOM.');
 });
diff --git a/test/qunit_test_error.js b/test/qunit_test_error.js
index fc19c53..0a87e3b 100644
--- a/test/qunit_test_error.js
+++ b/test/qunit_test_error.js
@@ -1,12 +1,12 @@
 
-test('basic test', function() {
-  expect(1);
-  ok(0, 'this had better work.');
+QUnit.test('basic test', function(assert) {
+  assert.expect(1);
+  assert.ok(0, 'this had better work.');
 });
 
 
-test('can access the DOM', function() {
-  expect(1);
+QUnit.test('can access the DOM', function(assert) {
+  assert.expect(1);
   var fixture = document.getElementById('qunit-fixture');
-  equal(fixture.innerText, 'this had better work.', 'should be able to access the DOM.');
+  assert.equal(fixture.innerText, 'this had better work.', 'should be able to access the DOM.');
 });

```

**Comment by jzaefferer** [comment]
That looks good. Could you provide that as a pull request?
